### DISCUSSION TITLE
Wanted Wake-Words

### DISCUSSION THREAD

**Discussion by fwartner** [fwartner/home-assistant-wakewords-collection#46] [Ideas]
Comment your wanted wake-words below this comment.

**Comment by kevvar** [comment]
Bro, this is awesome. I don't use to work with github usually, how can I upload some wake words to share? Maybe I can send you a google drive link with some wake words.

**Comment by fwartner** [comment]
Sure! Feel free to send me the link and i'm gonna add it to the repository!

**Comment by kevvar** [comment]
Great, here you go https://drive.google.com/drive/folders/1623ms9Uw7uj-5x1nH9Dwyp4XBW4w_hzJ
I'll be adding others words on the go

**Comment by fwartner** [comment]
Added the first ones! :)

**Comment by kevvar** [comment]
Great, I'm generating another ones in a couple of minutes.

**Comment by brewston** [comment]
Here's mine https://github.com/brewston/Homeaassistant-voiceintents/blob/main/edna.tflite

**Comment by fwartner** [comment]
Added :)

**Comment by kevvar** [comment]
Built today:

Kreitos, (yes, Kratos from the God of War game) disregard the writing, just spelling things
Mr. Wick (from the universe of John Wick)
Winston (from the universe of John Wick)
Maar vin, It sounds "Marvin"
Ok Jarvis
Hey Honey
Mr. Anderson (from the universe of Matrix)
Scarlett

**Comment by davefx** [comment]
I'm having some problems building new wake words.
I'd like to build "Hola, Casita", used in the Encanto movie.
To mimic the Spanish pronunciation, the string `oh laakaasee dah` sounds very similar to what it should sound in Spanish.

**Comment by kevvar** [comment]
> I'm having some problems building new wake words. I'd like to build "Hola, Casita", used in the Encanto movie. To mimic the Spanish pronunciation, the string `oh laakaasee dah` sounds very similar to what it should sound in Spanish.

I'll try it @davefx maybe it take a couple of hours because I'm using a large number of examples, to make a better model

**Comment by starsoccer** [comment]
Barclay from Tom Swift, https://nancydrew.fandom.com/wiki/Barclay

**Comment by kevvar** [comment]
@davefx I was trying to generate the wake word 3 times, each one gave me an error, checking the log it happens that the combination of letters causes an error in the generation of the model when the script looks for the pronunciations in the dictionaries and throws an error when executing the third training routine.

Today:

Ok Nabu
Ronaldo
Hey Homer
Skynet
Santana
Home Assistant
Barclay

@fwartner Could you please edit the names Kreitos to Kratos and Maar_vin to Marvin for a better understanding? I didn't think about it when uploaded

Edit: bro @davefx I made it work. I just used the phrase in spanish (believe me, it sounds good, I'm a native Spanish speaker). You have to say "Hola" a little fast and "casita" as it sounds. Find the file in the link above

**Comment by fwartner** [comment]
> @fwartner Could you please edit the names Kreitos to Kratos and Maar_vin to Marvin for a better understanding? I didn't think about it when uploaded

Will do later when i'm done with meetings.

**Comment by off112a** [comment]
This is fantastic 😀

I would love to have "Janet" as a wake word. It's from the TV serie The good place.

**Comment by kevvar** [comment]
> I would love to have "Janet" as a wake word. It's from the TV serie The good place.

Done, find it in the google drive link above and the repo when it has been updated

**Comment by fwartner** [comment]
Done :)

**Comment by sparkydave1981** [comment]
I know we already have 'hey Jarvis' but I'd like just 'Jarvis'.

EDIT:  I've just created this one so will add it as a PR.

EDIT again:  It's here in my repo until I get around to doing a PR.  I found it sounded better using 'djar vis' but it sounds like 'Jarvis'.
https://github.com/sparkydave1981/home-assistant_OpenWakeWord/blob/main/djar_vis.tflite

**Comment by itnassol** [comment]
Here is Ada if you want it.

https://github.com/itnassol/Ada

**Comment by thewolf-oz** [comment]
Have added a couple more and raised pull request - https://github.com/fwartner/home-assistant-wakewords-collection/pull/5

I'm having some troubles doing more, it keeps failing but I have a few I'm considering

**Comment by smartmatic** [comment]
Great idea, here is my contribution :-) Nothing special but still not yet listed in your collection.
https://drive.google.com/drive/folders/14viEMHMMgXi0aLMGLDWfNH2hwRpoiEtX?usp=sharing

**Comment by bluenazgul** [comment]
I Wish there would be wakewords for the German Community, sadly the Tool right now only Supports English

**Comment by smartmatic** [comment]
May be we can start collecting german wake words in a second issue until its possible to generate the files via collab?

**Comment by faduchesne** [comment]
thx for the list but I search for French wake word ;)

**Comment by smartmatic** [comment]
So we need a multilanguage sorted wake word list 😀Do somebody know when wake words can be translated to other languages?

**Comment by ijustlikeit** [comment]
ok boss

https://github.com/ijustlikeit/wake-words/blob/main/ok_boss.tflite

**Comment by kevvar** [comment]
> So we need a multilanguage sorted wake word list 😀Do somebody know when wake words can be translated to other languages?

Maybe on the next chapters

**Comment by kevvar** [comment]
Added new ones to drive https://drive.google.com/drive/folders/1623ms9Uw7uj-5x1nH9Dwyp4XBW4w_hzJ?usp=sharing

**Comment by pcwii** [comment]
Any chance I could get you to make 'kelsey' I managed to make a low sample model but things keep crashing when I try a high sample model. 'Kelsey' was an imaginary friend of my young daughter,  now that she's older she finds it neet to have kelsey talk back.

**Comment by fwartner** [comment]
Added new ones.

**Comment by fwartner** [comment]
> Any chance I could get you to make 'kelsey' I managed to make a low sample model but things keep crashing when I try a high sample model. 'Kelsey' was an imaginary friend of my young daughter, now that she's older she finds it neet to have kelsey talk back.

I am pretty sure that someone will make one. Since the collection is growing and growing, I am thinking about creating a website for this too where you can create requests for new wake-words.

**Comment by kevvar** [comment]
> Any chance I could get you to make 'kelsey' I managed to make a low sample model but things keep crashing when I try a high sample model. 'Kelsey' was an imaginary friend of my young daughter, now that she's older she finds it neet to have kelsey talk back.

Done https://drive.google.com/drive/folders/1623ms9Uw7uj-5x1nH9Dwyp4XBW4w_hzJ

**Comment by jimbob1001** [comment]
Would love Hey Artoo

**Comment by fwartner** [comment]
> > Any chance I could get you to make 'kelsey' I managed to make a low sample model but things keep crashing when I try a high sample model. 'Kelsey' was an imaginary friend of my young daughter, now that she's older she finds it neet to have kelsey talk back.
> 
> Done https://drive.google.com/drive/folders/1623ms9Uw7uj-5x1nH9Dwyp4XBW4w_hzJ

Already implemented.

**Comment by ijustlikeit** [comment]
hey home free

https://github.com/ijustlikeit/wake-words/blob/main/hey_home_free.tflite

Update: created pull as requested thanks.

**Comment by fwartner** [comment]
> hey home free
> 
> https://github.com/ijustlikeit/wake-words/blob/main/hey_home_free.tflite

Create a pull-request and i`m gonna merge it! :)

**Comment by Jacksaur** [comment]
Seeing as this Issue has been closed, does that mean new Wake Word suggestions are no longer being taken anymore?

Just in case they are, I'd love to have "Moon" (Rain World reference) and "Anubis" available!

**Comment by fwartner** [comment]
Sure.. They are open.. :)

**Comment by jacobkaad01** [comment]
Hey Genie and Ok Genie

**Comment by jacobkaad01** [comment]
> Hey Genie and Ok Genie

And maybe just “Genie” by itself?

**Comment by sayam93** [comment]
Could someone please do a "siri", not "hey siri"? The one I made using google colab doesn't seem to work...

**Comment by Greylinux** [comment]
I'm really surprised Cortana hasn't been requested , is there a reason behind that ?

**Reply by loryanstrant** [reply]
I'd love for Cortana as well, especially as my kids actually requested that be the wake word we use and persona of our system.
We had the Hartman Kardon Invoke speaker so they got used to it. :-)

**Comment by sayam93** [comment]
> Comment your wanted wake-words below this comment.

Could someone please do a "siri", not "hey siri"? The one I made using google colab doesn't seem to work!

**Comment by Dgreendow** [comment]
I would love "Hey Bozo"

**Comment by qJake** [comment]
Can you add "**Artemis**"? It's a nice, unique, three-syllable word that isn't often spoken - it makes for a great wake word. 🙂

**Comment by sayam93** [comment]
Could someone please do a "siri", not "hey siri"? The one I made using google colab doesn't seem to work...

**Comment by Hedda** [comment]
Submitting request here as sounds like this wakeword might not be added as official default option in upstream for unknown reasons -> https://github.com/esphome/micro-wake-word-models/issues/11

Please consider including a wake word for 'Assist' as part of a combination wakeword, specifically "Hey Assist" and "Okay Assist".

- _**Hey Assist.**_

and 

- _**Okay Assist.**_

As in

- "_**Okay Assist, turn on the dining room lights**_".

And with the word "Assist" I am here referring to Home Assistant's Assist for the official Assist conversation agent and pipeline in Home Assistant:

* https://www.home-assistant.io/voice_control/

The new official Home Assistent voice assistant integration has now also been named Assist Satellite

* https://www.home-assistant.io/integrations/assist_satellite/

and so is the Assist satellite entity

* https://developers.home-assistant.io/blog/2024/10/01/assist-satellite-entity/
  * https://developers.home-assistant.io/docs/core/entity/assist-satellite/
    * https://github.com/home-assistant/architecture/discussions/1136
      * https://github.com/home-assistant/core/pulls?q=is%3A+assist_satellite%22

After all, Home Assistant's official voice assistant is called "Assistant", and there are even official logos and branding for that name.

![image](https://github.com/user-attachments/assets/4d4b4b64-8382-4142-97c9-850cd2274b3f)

**Describe the solution you'd like**

Add wakewords

- _**Hey Assist.**_

and 

- _**Okay Assist.**_

**Describe alternatives you've considered**

My reasons for the request is partially for the same as in https://github.com/esphome/micro-wake-word-models/issues/8 which are that 'Nabu' / 'Nabucasa' does not have same brand recognition. Nabu as in Nabu Casa is also only the official partner to Open Home Foundation and the ESPHome and Home Assistant projects, so it does not make much sense of it being a default.

Would be great with option to use 'Assist' in wake word combos as a way to begin to add some more "brand recognition" for that feature.

Even on Nabu Casa's website it says that "Assist" is the official name that the voice assistant, so makes no sence "Nabu" being default?

**Additional context**

Again, I also submitted this as a feature request upstream as wish it was included by default but sounds as if it probably will not -> https://github.com/esphome/micro-wake-word-models/issues/11

**Comment by johnboiles** [comment]
I'd love just a simple "Home Assistant" wake word. Allows me to advertise HA to my guests. "Home Assistant turn on the dining room lights"

**Reply by endlesvyd** [reply]
> I'd love just a simple "Home Assistant" wake word. Allows me to advertise HA to my guests. "Home Assistant turn on the dining room lights"

I trained a more accurate "home assistant", since the previous model was giving me too many false activations. 

https://1drv.ms/u/c/a90bd5462ad0fef6/EbXfphVKy15Gk83JWegRIGABvXizzIED-1lbXCjgwXc59A

**Reply by endlesvyd** [reply]
@fwartner , any chance you could add this one to the repository as a v2? I've had much better response rates with it than the one thats currently in the repository. I'm not much of a github user though so not sure how to add it myself.

**Comment by Hedda** [comment]
FYI, the training process to create your own microWakeWord (mWW) wake words has now been and documented (and is also open source): 

- https://github.com/kahrendt/microWakeWord?tab=readme-ov-file#model-and-training-design-notes

Unfortunatly the current process is very complicated and there is not yet any all-in-one training notebook like there is for openWakeWord :

- https://github.com/kahrendt/microWakeWord/issues/2
  - https://github.com/kahrendt/microWakeWord/issues/29

PS: microWakeWord is a open-source wake word engine that can run on microcontrollers such as the ESP32 family (like the ESP32-S3 series):

- https://github.com/kahrendt/microWakeWord

PPS: Currently there is a requirement is that the microcontroller features support for TensorFlow AI/ML hardware acceleration via LiteRT:

- https://ai.google.dev/edge/litert/microcontrollers/overview

**Comment by AndreKR** [comment]
We're currently using "Americano" (from Porcupine) and I'd like to keep that, so please: Americano

**Comment by RickSmitth** [comment]
Hey there thank you for all your work, amazing to find such pretrained wakewords here.

Im having trouble by traning my own word in the model trainer because it always run into error in step 3. but have no clue why. 
can someone create the wakeword Hey GPT for me?
i had best results in pronounciation with "Hey__Geepee_tee"

Thank you very much
BR 
RickSmitth

**Comment by ralbright-wa** [comment]
I have 3 requests... and for WHATEVER reason I cannot get the colab notebook to run them correctly.
Skippy (from the Expeditionary Force Book series)
Alliance (working on my RV)
Paradigm (working on my RV)

**Reply by orionpt** [reply]
+1 for Skippy - the Magnificent :)

**Comment by razvanab** [comment]
My WakeWord

https://github.com/razvanab/WakeWord.git

**Reply by fwartner** [reply]
Create a Pull-Request and i´ll merge it.

**Reply by razvanab** [reply]
I tried to do it but I didn't figure out how, and I don't want to break something on this repo.

**Comment by iambendick** [comment]
Could I request Echo
The alternate word that Alexa devices use?

**Comment by Tsamsiyu1** [comment]
I would love to request "Hey Skippy"

**Comment by fwartner** [comment]
Would be great if someone creates the requested Wake-Words.. :)

**Comment by SdeGeata** [comment]
Hello everyone;

Requesting an openWakeWord model for the phrase ‘to whit to whoo’ (like an owl hooting). For use with a Wyoming Satellite on a Home Assistant security owl project. My computer is too damn old to run the wake word training myself. Thanks for your help!

**Comment by Matte982** [comment]
Hi everyone,
I'm looking for a custom wake word model for "Alfred" specifically trained for Italian phonetics (clear "R" and "E" sounds).
Most existing models for Alfred are trained on English datasets and they don't work reliably with an Italian accent unless we "fake" an American pronunciation, which isn't ideal for daily use.
Details:
Wake Word: Alfred
Language: Italian (it-IT)
Format: .tflite (for openWakeWord)
Hardware: Home Assistant Green
Prefix: None (just the name)
If someone with a training environment could generate this using Italian Piper synthetic voices (or high-quality Italian samples), it would be a huge help for the Italian HA community.
Thank you! 😁😁

**Comment by DanHa5442** [comment]
Poprosím o probouzecí slovo: Matyldo/Matylda

**Comment by vascofo** [comment]
I'm trying to get a pt-PT "Nio" - _Nee-oh_ - similar to the "hey neo" but without the hey - any chance?
